src/libs/ReportUtils.js

```diff
@@ -417,12 +433,49 @@ function isPolicyExpenseChatAdmin(report, policies) {
     return policyRole === CONST.POLICY.ROLE.ADMIN;
 }
 
+/**
+ * Returns true if report has a parent and is therefore a Thread.
+ *
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function isThread(report) {
+    if (!report || !report.parentReportID || !report.parentReportActionID) {
```

Comment:
i find the inverted logic harder to read. How about switching it like so,
```
if (report && report.parentReportActionID) {
    return true;
}
return false;
```

Also you can actually just simplify this to one line i.e.,
```
return _.has(report, 'parentReportActionID');
```
it should return false if report is undefined anyway making it safe i believe.